app/metro.config.js

```diff
@@ -1,7 +1,6 @@
 /**
  * @file Metro configuration for React Native
  * https://github.com/facebook/react-native
- *
```

Comment:
> I think it has to go, and I think I've touched this file already in a different commit? There was one where I deleted the format tag that comes just after line 4 in this file, and then once I'd done that the linter started complaining about this trailing "*" on line 4, so I deleted it here. A) does that make sense or am I getting confused with something here, and B) in future is it more helpful to roll up all these smaller linting fixes into a single commit?

Ok no worries if the linter's being difficult

Reply:
What editor are you using @vglampard ?

Reply:
ESlint doesn't like empty lines in the comment block. When @joefhall added the @file in January the CI script was happy because the additional line was the only change and passed the test. The rest of the comment was Jamie in November 2021 before we added Linting. Now it's looking at the empty line and having a moan.